goal_src/kernel/gkernel-h.gc

```diff
@@ -34,11 +34,11 @@
 
 ;; the size of the execution stack (~14 kB) shared by all threads
 ;; OpenGOAL NOTE: increased to 32kB
-(defconstant DPROCESS_STACK_SIZE #x8000)
+(defconstant DPROCESS_STACK_SIZE #x10000)
```

Comment:
update comment to say 64 kB

Reply:
i walked these back, since they weren't actually required